src/pages/monthly-plan.js

```diff
@@ -62,6 +52,24 @@ const suggestedAmounts = [
   },
 ];
 
+const getProfileData = (id, accessToken, excludedRepos) => {
```

Comment:
This looks duplicated with the code in `fetch.js`

Reply:
Oops that should not be there again, I thought I removed it.